client/blocks/reader-post-card/featured-image.jsx

```diff
@@ -8,14 +8,15 @@ import { noop } from 'lodash';
  * Internal Dependencies
  */
 import cssSafeUrl from 'lib/css-safe-url';
+import resizeImageUrl from 'lib/resize-image-url';
 
 const FeaturedImage = ( { imageUri, href, children, onClick } ) => {
 	if ( imageUri === undefined ) {
 		return null;
 	}
 
 	const featuredImageStyle = {
-		backgroundImage: 'url(' + cssSafeUrl( imageUri ) + ')',
+		backgroundImage: 'url(' + cssSafeUrl( resizeImageUrl( imageUri, { w: 250 } ) ) + ')',
```

Comment:
Worth sticking the `250` into a const and explaining why we're using that value?